src/lib/local-storage.js

```diff
@@ -360,6 +360,15 @@ class LocalStorage implements IStorage {
       name,
       (data, cb) => {
         if (data._attachments[filename]) {
+          // get version form _attachments;
+          // there aways had 'version' in verdaccio publish package.
+          // but there was no 'version' in package witch unlink form npmjs
```

Comment:
form => from
witch => which

What is this line about?

Reply:
// [1] get version from _attachments;
// [2] there aways had 'version' in data._attachments[filename] which developer publish the package to verdaccio.
// [3] but there was no 'version' in data._attachments[filename], if the package uplink(pull) form source registry like 'www.npmjs.com'
 
for example: 
I fork react 16.0.0 and make a new version called 16.1.0.
then I publish it to my private verdaccio.
verdaccio will pull react from source registry.
If I unpublish react@16.1.0, it's the case above [2]
If I unpublish react@16.0.0, it's the case above [3]

Reply:
I think we should completely remove these comments (they are too verbose) and only describe that we remove the relevant parts on `unpublish`.